frontends/web/src/components/aopp/verifyaddress.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import * as accountAPI from '../../api/account';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { Button } from '../forms';
+import { WaitDialog } from '../wait-dialog/wait-dialog';
+
+interface State {
+    verifying: boolean;
+}
+
+interface VerifyAddressProps {
+    accountCode: string;
+    address: string;
+    addressID: string;
+}
+
+type Props = VerifyAddressProps & TranslateProps;
+
+class VerifyAddress extends Component<Props, State> {
```

Comment:
Nit: either

```js
public readonly state: State = {
        verifying: false,
};
```

or make `verifying` optional in the State interface

Reply:
why? it is false by default.

Reply:
Where is it false by default?
It looks `undefined`